interceptors/options.go

```diff
@@ -30,3 +31,14 @@ func WithHystrixName(name string) clientOption {
 		},
 	}
 }
+
+// WithHystrixErrorSuppressor applies a function that you can write logics to suppress errors to hystrix interceptor
+func WithHystrixErrorSuppressor(e func(err error) error) clientOption {
```

Comment:
Actually, we have some Hystrix options in https://github.com/carousell/go-utils/blob/master/utils/hystrixwrapper/hystrix_opts.go. I think it might be good if we can utilise this function here.

Reply:
https://github.com/carousell/go-utils is a private repo. Can't be use here.